bin/collator/src/parachain/service.rs

```diff
@@ -326,10 +326,6 @@ where
         bool,
     ) -> Result<Box<dyn ParachainConsensus<Block>>, sc_service::Error>,
 {
-    if matches!(parachain_config.role, Role::Light) {
```

Comment:
Removing this means we got LC configuration part somewhere?

Reply:
I don't know where it is, all I've found is that it was removed from substrate.
To do the uplift, I removed the enum mention.

Reply:
This shouldn't impact other types of nodes. We'd like to keep up with the LightClient development situation on Substrate side I think.